backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/parsers/SPDXParserTools.java

```diff
@@ -135,7 +135,7 @@ private static String getLocalName(String label) {
                             return path.substring(path.lastIndexOf('/') + 1);
                         }
                     } catch (UnsupportedEncodingException | MalformedURLException e) {
-                      return "";
+                        return "";
```

Comment:
Please remove the extra space

Reply:
not needed, I fixed the incorrect formatting.